src/sdk/namespace_fs.js

```diff
@@ -1118,7 +1118,9 @@ class NamespaceFS {
         const fs_context = this.prepare_fs_context(object_sdk);
         const open_mode = 'w';
         try {
-            let MD5Async = config.NSFS_CALCULATE_MD5 ? new (nb_native().crypto.MD5Async)() : undefined;
+            const namespace_bucket_config = await object_sdk.read_bucket_sdk_namespace_info(params.bucket.name)
```

Comment:
I don't like that we need to call this inside the namespace code itself.
I would expect that object_sdk would handle it and pass the option inside the params.